docs/api/Group.md

```diff
@@ -18,6 +18,7 @@ A group of layers. It is also an instance of [Layer](#layer) so all the methods
 | frame<span class="arg-type">[Rectangle](#rectangle)</span> | The frame of the Group. This is given in coordinates that are local to the parent of the layer. |
 | flow<span class="arg-type">[Flow](#flow)</span>            | The prototyping action associated with the Group.                                               |
 | style<span class="arg-type">[Style](#style)</span>         | The style of the Group.                                                                         |
+| sharedStyleId<span class="arg-type">string / null</span>   | The ID of the [SharedStyle](#sharedstyle) this Group is linked to if any.                       |
```

Comment:
Should this have a note about having to call `layer.style.syncWithSharedStyle(sharedStyle)` somewhere? Would be nice to not have to repeat these across all StyleLayer subclasses 😕

Reply:
Anything about `sharedStyle` or just the `sharedStyleId`??

Reply:
> note about having to call layer.style.syncWithSharedStyle(sharedStyle) somewhere

Yeah probably, not sure where. Maybe in the Style doc 🤔 

> Anything about sharedStyle or just the sharedStyleId??

That's a good question. `sharedStyle` is not enumerable, meaning that is you do `console.log(layer)`, it won't show `sharedStyle`. There is also the `parent` property and a few others. I haven't documented any of them yet, we probably need another section for each document.

Feel free to send a PR, we wanted to merge it quickly to include it in Sketch 52.1